build.sh

```diff
@@ -100,11 +100,13 @@ if [ ! -f "$HOME/.cache/bin/cppcheck" ]; then
   
   wget http://llvm.org/releases/3.6.2/clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04.tar.xz
   tar xvf clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04.tar.xz
-  cp -R clang+llvm-3.6.2-x86_64-linux-gnu/* $HOME/.cache
+  
+  # copy, not move, since .cache may contain other files in subdirs already
+  cp -R clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04/* $HOME/.cache
   
   # remove directories, otherwise copyright-header check complains
   rm -rf ./cppcheck
-  rm -rf ./clang+llvm-3.6.2-x86_64-linux-gnu
+  rm -rf ./clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04
 fi
 
 export PATH=$PATH:$HOME/.cache/bin
```

Comment:
I still think the new path should be added to the head of PATH, not the tail:

```bash
export PATH=$HOME/.cache/bin:$PATH
```

This is what the build output from travis says:
```bash
echo $PATH

/home/travis/bin:/home/travis/.local/bin:/home/travis/.rvm/gems/ruby-2.2.3/bin:/home/travis/.rvm/gems/ruby-2.2.3@global/bin:/home/travis/.rvm/rubies/ruby-2.2.3/bin:/home/travis/.pyenv/shims:/home/travis/.phpenv/shims:/home/travis/.php-build/bin:/home/travis/.local/bin:/home/travis/.pyenv/shims:/opt/pyenv/bin:/home/travis/.phpenv/shims:/home/travis/perl5/perlbrew/bin:./node_modules/.bin:/usr/local/clang-3.5.0/bin:/home/travis/.local/bin:/home/travis/.pyenv/shims:/opt/pyenv/bin:/home/travis/.phpenv/shims:./node_modules/.bin:/usr/local/clang-3.5.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/travis/.phpenv/bin:/home/travis/.phpenv/bin:/home/travis/.rvm/bin:/home/travis/.rvm/bin
```

It has clang-3.5 there already in the path, this version will be invoked if we append to the tail of PATH, right?